config/initializers/extends.rb

```diff
@@ -41,6 +41,25 @@ class Extends
                            my_module_repository: 17,
                            my_module_protocol: 18 }
 
+  EXPORT_ALL_PROJECT_ELEMENTS = {
```

Comment:
So it is not possible to have here all needed arguments for 'gen_element_content' method?

Reply:
Elements not consistent. In some cases, we iterate and pass only one object, in other, we pass parent object and associate objects. And special cases for results and assigned repositories.